src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+- [Accessibility](#accessibility)
+
+<span id="html-elements" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayTwo"
+>
+  HTML Elements
+</SprkHeading>
+
+HTML (Hyper Text Markup Language) consists of series of elements, which enclose, or wrap pieces of content they appear or act a certain way.
+
+Elements describe or represent the meaning, structure, or style of a word, to:
+
+- Web browsers
+- Developers
+- Users of assistive technologies
+
+> For more information, visit <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element"> Modzilla's Developer Documentation - HTML elements</a>.
+
+As a UX Writer, it's important to:
+
+- **Understand** what is the purpose of the enclosed text.
+- **Know** what element to use to use depending on the purpose of the text.
+- **Ensure** we are communicating the intended meaning of the enclosed text to the users.
+
+<span id="creating-elements-using-tags" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h4"
+  additionalClasses="sprk-u-mbs sprk-u-Measure"
+  variant="displayFour"
+>
+  Creating Elements Using Tags
+</SprkHeading>
+
+Use **tags** to create an element. Tags delimit the start and end of elements in the markup.
+
+An element consists of three parts:
+
+- The **opening tag** that represents the start of the paragraph element.
+- The **content** enclosed inside the tags.
+- The **closing tag** that represents where the element ends.
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Example
+</SprkHeading>
+
+`<p>` Download the Rocket Homes mobile application on your mobile device to start looking for your dream home.`</p>`
+
+- The `<p>` represents the start of the paragraph element.
+- The content is the enclosed text (<i>"Download the Rocket.... " </i>) inside the tags.
+- The `</p>` represents where the element ends.
+
+#### Common HTML Elements For UX Writers
+
+Use the strong and emphasis elements, to ensure users can perceive the semantic meaning indicated by the visual treatment, even if they’re using assistive technology like a screen reader.
+
+- The **strong element** `<strong></strong>`, defines text with strong importance.
+- The **emphasis element** `<em></em>`, defines text with stressed emphasis.
+
+> Use the **bold element** `<b></b>` and the **italic or idiomatic text element** `<i></i>` for formatting and to display special types of text.
+
+- The **bold element** `<b></b>` , draws attention to the element's contents, without extra importance.
+
+- The **italic or idiomatic text element** `<i></i>`, defines a part of text in an alternate voice or mood.
+
+<span id="semantic-elements" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayTwo"
+>
+  Semantic Elements
```

Comment:
Rename this to something like:

`Communicating Importance or Emphasis`

Why:
- We've already explain what semantic elements are several times in this, we're repeating information.
- This is what the actual section is explaining. Strong and Emphasis